packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/resiliation/resiliation.routing.js

```diff
@@ -62,6 +71,20 @@ export default /* @ngInject */ ($stateProvider) => {
           type: 'action',
         });
       },
+      goToDeleteService: /* @ngInject */ (
+        $window,
+        coreURLBuilder,
+        serviceName,
+      ) => () => {
+        set(
+          $window.location,
+          'href',
+          coreURLBuilder.buildURL(
+            'dedicated',
+            `#/billing/autorenew/delete?serviceId=${serviceName}`,
```

Comment:
Can we go to the state `app.account.billing.autorenew.delete` instead of redirecting to a url?

Reply:
The navigation is to a different module. coreURLBuilder builder might be a better choice here.